src/views/GrowersView.js

```diff
@@ -19,12 +20,14 @@ function GrowersView() {
       style={{ flexWrap: 'nowrap', height: '100%', overflow: 'hidden' }}
     >
       <GrowerProvider>
-        <SpeciesProvider>
-          <TagsProvider>
-            <GrowerFilterHeader />
-            <Growers />
-          </TagsProvider>
-        </SpeciesProvider>
+        <CapturesProvider>
```

Comment:
You don't need to go through the CapturesContext for this change – just call the appropriate treeTrackerApi method directly, or through GrowerContext.